.gitignore

```diff
@@ -18,3 +18,4 @@ out/
 .metadata/
 .recommenders/
 *.log
+/.nb-gradle/
```

Comment:
To be consistent with the other entries, can we just make it `.nb-gradle`? Also, should we add `.nb-gradle-properties`?